testsuite/ui/views/admin/product/application.py

```diff
@@ -15,6 +16,20 @@ class ApplicationPlansView(BaseProductView):
         "Name": Text("./a")
     })
     create_button = ThreescaleCreateButton()
+    change_plan_button = ThreescaleSubmitButton()
+    default_plan_select = Select()
+    notification = View.nested(FlashMessage)
+
+    def change_default_plan(self, plan_name: str):
```

Comment:
Why you add this function? You are not using it anywhere in the test.

Reply:
I have planned to add an extended test, that changes the default plan but after discussion, it will be part of the next PR. So this code only enables the basic functionality of changing the default plan.

Reply:
So it's not in scope of this PR and should be part of the next PR right?

Reply:
Removed